gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java

```diff
@@ -107,6 +104,10 @@ public Orchestrator(Config config, Optional<TopologyCatalog> topologyCatalog, Op
         | ClassNotFoundException e) {
       throw new RuntimeException(e);
     }
+    //Pass the topologySpecMap to the DagManager
```

Comment:
Shall we move this block under line 93?

Reply:
We need this after the spec compiler initialization, to make sure that the topologySpecMap has been instantiated. Added comment to make it explicit.